cli/tests/test_diff_command.rs

```diff
@@ -106,6 +113,15 @@ fn test_diff_basic() {
     @@ -1,0 +1,1 @@
     +foo
     "###);
+
+    let stdout = test_env.jj_cmd_success(&repo_path, &["diff", "--stat"]);
+    let stdout = strip_lines_prefix(stdout, " ");
```

Comment:
Why strip a prefix? I'd prefer to see the actual output in the test, unless that causes problems for some reason

Reply:
I fiddled with the `r###"` format and could not get it to *not* strip the leading whitespace so I ended up stripping it from the output. If you have a better idea I'm all ears.

Reply:
Ah, I see, so it's related to that other comment I had about a leading space then. Hopefully we can drop that leading space and not have this problem.

Reply:
Done.

Reply:
Thanks! All looks good to me now. Feel free to mark this resolved and enable auto-merge.